src/UI/Component/Input/Container/Factory.php

```diff
@@ -29,4 +29,71 @@
 	 * @return  \ILIAS\UI\Component\Input\Container\Form\Factory
 	 */
 	public function form();
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Filters are used to let the user limit content within a table, list or any other collection of items
+	 *      presented on the screen.
+	 *   composition: >
+	 *      Filters are composed of two visually separated entities:
+	 *      First, there is the Filter Bar at the top. It contains an Expand/Collapse Glyph on the left side
+	 *      On the right, an "Apply" Bulky Button, a "Reset" Bulky Button and a Toggle Button are placed. An additional
+	 *      row with set filter settings will be shown at the bottom of the Filter Bar, when the Filter Bar is collapsed
+	 *      and the Filter is activated at the same time.
+	 *      Second, there is an area where so-called Filter Fields are displayed. Every Filter Field holds a Label on the left
+	 *      and a "Remove" Glyph on the right. Every Filter Field stores an Input Field. After the last Filter Field,
+	 *      an "Add" Bulky Button is always shown.
+	 *   effect: >
+	 *      In the Filter Bar:
+	 *      Clicking on the Expand/Collapse Glyph expands/collapses the second entity of the Filter,
+	 *      where the Filter Fields are placed. When the Filter Bar is expanded, the row with set filter settings
+	 *      at the bottom will be hidden.
+	 *      Clicking on the "Apply" Bulky Button applies the settings which the user has made for the Filter and
+	 *      reloads the page immediately.
+	 *      Clicking on the "Reset" Bulky Button resets the Filter to the initial state and reloads the page immediately.
```

Comment:
Maybe change reloads the page immediatly to "resets the content of the operated container (e.g. Table) immediatly"